data-prepper-api/src/main/java/com/amazon/dataprepper/model/processor/AbstractPrepper.java

```diff
@@ -46,5 +46,5 @@ public AbstractPrepper(final PluginSetting pluginSetting) {
      * @param records Input records
      * @return Processed records
      */
```

Comment:
I asked this question to Technical Writer but haven't got any response - Are we changing the name of the component from `Processor` to `Prepper` ? [just for my information, I would like to leave it as Processor but ok with Prepper]

Reply:
I think we should keep it as `AbstractProcessor` - but I am fine if everyone wants to update it to `AbstractPrepper`

Reply:
I am fine either way, lets just all align on one name

Reply:
https://github.com/opendistro-for-elasticsearch/Data-Prepper/issues/209 :)